xml/docu_styleguide.structure.xml

```diff
@@ -302,6 +302,55 @@ default 0 <co xml:id="co-default"/></screen>
     </tbody>
    </tgroup>
   </table>
+
+  <para>You can group different callouts to point to the same annotation
+  so that it is not repeated. To do this, complete the following steps:</para> 
+  <orderedlist>
+    <listitem>
+     <para>
+      Create the <tag class="emptytag">co</tag> element with the
+      <tag class="emptytag">xml:id</tag> attribute to comment on the first line.
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      On another line, use the <tag class="emptytag">xref</tag> element
+      to point to the first line.
```

Comment:
To be precise, you don't point to the "first line". You point to an ID that is a co element:

```suggestion
      to point to the ID from the previous step.
```